### ISSUE TITLE
Different height select box and input box

### ISSUE THREAD

**Issue by lukaspijak** [Dogfalo/materialize#546]
![selects](https://cloud.githubusercontent.com/assets/2571714/5944945/42d3d0ba-a72b-11e4-808b-89f6012723aa.png)

```html
       <div class="row">
           <div class="col s12 m12 l6">
                <div class="input-field">
                    {input company_name}
                    {label company_name/}
                </div>
                <div >
                    {label gender /}
                    {input gender}
                </div>
                <div >
                    {label status/}
                    {input status}          
                </div>
                <div class="input-field">
                    {input birth_date}
                    {label birth_date /}
                </div>
            </div>
            <div class="col s12 m12 l6">
                <div class="input-field">
                    {input company_name}
                    {label company_name/}
                </div>
                <div class="input-field">
                    {input company_number}
                    {label company_number/}
                </div>
                <div class="input-field">
                    {input company_vat}
                    {label company_vat/}
                </div>
                <div class="input-field">
                    {input company_id_dph}
                    {label company_id_dph/}
                </div>
            </div>
        </div>
```

**Comment by wwalmnes** [comment]
I managed to get it working with some small fixes: 
![screen shot 2015-01-29 at 09 52 05](https://cloud.githubusercontent.com/assets/1506750/5954797/8e8c2874-a79c-11e4-81d4-79e0d329fc4e.png)
1: Add the class "input-field" to the div containing the select tag. 
2: Seems like the input tags uses "box-sizing:content-box" while the span (".select-dropdown") uses "box-sizing:border-box". If you change ".select-dropdown" to content-box you get the correct size.
3: I think the label should be avoided. Just add a gray color to the option value disabled.
These three steps gave me the result shown above.

If this is acceptable I can do a pull request.

**Comment by acburst** [comment]
Should be fixed in ae38b2907af92f310a3df08c9542ae48e6468b7a

**Comment by lukaspijak** [comment]
can i get example code?
